lib/vdsm/v2v.py

```diff
@@ -231,6 +231,11 @@ def get_ova_info(ova_path):
     ns = {'ovf': _OVF_NS, 'rasd': _RASD_NS}
 
     ovf_str = _read_ovf_from_ova(ova_path)
+    ovf_str_list = ovf_str.split(_OVF_NS)
+    ovf_str = ovf_str_list[0] + str(_OVF_NS) + '"' + '\n' + 'xmlns=' + \
+              '"' + str(_OVF_NS) + ovf_str_list[1]
+    extract_ns = _extract_ns_map(ovf_str)
+    ns = extract_ns if extract_ns else ns
```

Comment:
This does not look like the right way to support namspaces, but I'm not sure what is the problem you are trying to solve. Can you explain the issue you are trying to solve by this change?

Do you have some failures with certain ova files? Can you share the OVF that reproduce the failure?

Reply:
<ns1:Envelope
    xmlns:ns1="http://schemas.dmtf.org/ovf/envelope/1"
    xmlns:ns4="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData"
    xmlns:ns3="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData">
    <References>
        <File ns1:id="vmdisk1" ns1:href="IPv6-1" ns1:size="1048576"/>
    </References>
    <DiskSection>
        <Info>Disk</Info>
        <Disk ns1:diskId="vda" ns1:fileRef="vmdisk1" ns1:capacity="85899345920" ns1:format="qcow2"/>
    </DiskSection>
    <NetworkSection>
        <Network ns1:name="vswitch0"/>
    </NetworkSection>
    <VirtualSystem ns1:id="CAS">
        <Name>IPv6-1</Name>
        <OperatingSystemSection>
            <Description>Red Hat Enterprise Linux 7(64-bit)</Description>
        </OperatingSystemSection>
        <VirtualHardwareSection ns1:transport="iso">
            <System>
                <ns3:VirtualSystemType></ns3:VirtualSystemType>
            </System>
            <Item>
                <ns4:AllocationUnits>hertz * 10^6</ns4:AllocationUnits>
                <ns4:ElementName>CPU</ns4:ElementName>
                <ns4:InstanceID>1</ns4:InstanceID>
                <ns4:Reservation>0</ns4:Reservation>
                <ns4:ResourceType>3</ns4:ResourceType>
                <ns4:VirtualQuantity>2</ns4:VirtualQuantity>
                <ns4:CoresPerSocket>1</ns4:CoresPerSocket>
            </Item>
            <Item>
                <ns4:AllocationUnits>byte * 2^20</ns4:AllocationUnits>
                <ns4:ElementName>memory</ns4:ElementName>
                <ns4:InstanceID>2</ns4:InstanceID>
                <ns4:ResourceType>4</ns4:ResourceType>
                <ns4:VirtualQuantity>4096</ns4:VirtualQuantity>
                <ns4:CoresPerSocket>0</ns4:CoresPerSocket>
            </Item>
            <Item>
                <ns4:Connection>vswitch0</ns4:Connection>
                <ns4:ElementName>Ethernet adapter 0</ns4:ElementName>
                <ns4:InstanceID>3</ns4:InstanceID>
                <ns4:ResourceType>10</ns4:ResourceType>
                <ns4:ResourceSubType>virtio</ns4:ResourceSubType>
                <ns4:VirtualQuantity>0</ns4:VirtualQuantity>
                <ns4:CoresPerSocket>0</ns4:CoresPerSocket>
            </Item>
            <Item>
                <ns4:Address>vda</ns4:Address>
                <ns4:ElementName>virtio</ns4:ElementName>
                <ns4:InstanceID>4</ns4:InstanceID>
                <ns4:ResourceType>17</ns4:ResourceType>
                <ns4:VirtualQuantity>0</ns4:VirtualQuantity>
                <ns4:CoresPerSocket>0</ns4:CoresPerSocket>
            </Item>
        </VirtualHardwareSection>
    </VirtualSystem>
</ns1:Envelope>